src/DisplayBoard.cpp

```diff
@@ -101,20 +116,73 @@ std::string drawColumns(const unsigned int maxElementsInRows,
     for (int i = heightCol; i > 0; --i)
     {
         columns += PADDING + drawPadding(maxElementsInRows);
-        columns += drawEmptyLine(sizeCols);
+        columns += VERTICAL;
+        columns += drawRowOfColumns(i,stringClues);
+        columns += VERTICAL;
         columns += "\n";
     }
+    return columns;
+}
 
+std::vector<std::string> getFormattedColumns(const unsigned int sizeCols,
+                                              const Lines cluesCols)
+{
+    std::vector<std::string> stringLines {};
+    if(cluesCols.empty() == false)
+        std::for_each(cluesCols.begin(), cluesCols.end(),
+                      [&](const auto line)
+        {
+            std::string strline = {};
+            std::for_each(line.begin(), line.end(),
+                         [&](const auto i)
+            {
+                strline += std::to_string(i);
+            });
+            if(strline.size() < sizeCols)
+                while (strline.size() != sizeCols) {
+                    strline.push_back(' ');
+                }
+            stringLines.emplace_back(strline);
+        });
+    return stringLines;
+}
+
+std::string drawRowOfColumns(const unsigned pos,
+                       std::vector<std::string> clues)
+{
+    std::string columns {};
+    std::string s = " ";
+    if(clues.empty() == false)
+    for(auto line : clues)
+        columns += s + line.at(pos-1);
     return columns;
 }
 
-std::string drawRow(const unsigned int maxElementsInRows)
+std::string drawRow(const unsigned int maxElementsInRows, Line row)
 {
     std::string rows;
 
     rows += VERTICAL;
-    rows += drawPadding(maxElementsInRows);
-
+    if(row.empty() == true)
+    {
+        rows += drawPadding(maxElementsInRows);
+    }else
+    {
+        if(row.size()< maxElementsInRows)
+        {
+            for(int i = maxElementsInRows - row.size(); i > 0; i--)
+            {
+                rows.push_back(PADDING);
+                rows.push_back(PADDING);
```

Comment:
maybe instead of calling push_back two times we can define something like DOUBLE_PADDING?